src/yb/util/CMakeLists.txt

```diff
@@ -207,12 +207,12 @@ set(UTIL_SRCS
   ntp_clock.cc
   oid_generator.cc
   once.cc
+  operation_counter.cc
```

Comment:
It looks like there is some issue with the way this branch has included other changes with your change. Might be worth doing a rebase or at worst, a force push to remove these additional changes.